login.html

```diff
@@ -7,5 +7,6 @@
 </head>
 <body>
     <h1>Login work stated</h1>
+    <h2>Initial login with email and password done</h2>
```

Comment:
write the proper function name

Reply:
ok got it. i am working on it

Reply:
function name corrected please review this

Reply:
ok you are not good to go